### DISCUSSION TITLE
Anime Model config

### DISCUSSION THREAD

**Discussion by haydn-jones** [xinntao/Real-ESRGAN#93] [Q&A] [answered]
Does anyone know where I can find the configurations used to train the Anime model that was released? Like the yaml files in `options/`. Are they the same as the real image configurations?

**Comment by xinntao** [comment, marked as answer]
@haydn-jones 
The training config for the anime model is the same as the real image config, except:
1. used dataset
2. block number of networks

**Reply by haydn-jones** [reply]
Thanks!
